### PR TITLE
Remove helpers

### PR DESCRIPTION
**Pull request by steakunderscore**

### CODE REVIEW

**Comment by mathieugagne** [comment]
All good! :+1:
Just needs a rebase

File: app/models/job.rb [hunk 1]
```diff
@@ -13,7 +15,5 @@ class Job < ActiveRecord::Base
   validates :description,
             presence: true,
             length: { maximum: MAX_TEXT_COLUMN_LENGTH }
-  validates :state,
-            presence: true,
-            inclusion: { in: STATES }
+  validates :state, presence: true
```
Comment (sophiedeziel) [inline app/models/job.rb]: We don't validate inclusion anymore?

**Comment by sophiedeziel** [comment]
I just noticed that this pull request was opened from a branch that is based on another branch that has a pull request opened too... Was this an accident or you opened a PR for changed asked in the previous one?

**Comment by steakunderscore** [comment]
@sophiedeziel I think it's just me being noob with githubs pushing/reviewing model. I'm use to one patch per review. I was waiting for https://github.com/montrealrb/Montreal.rb/pull/180 to be merged, but needed it for this patch too.

**Comment by sophiedeziel** [comment]
Are you on the montreal.rb slack team? I can spend some time with you :smile: 
You can request an invite here: https://montrealrb-slack.herokuapp.com/

**Pull request merged**